client/src/javascript/components/general/form-elements/TextboxRepeater.js

```diff
@@ -6,7 +6,7 @@ import RemoveMini from '../../icons/RemoveMini';
 
 export default class TextboxRepeater extends React.PureComponent {
   state = {
-    textboxes: [{id: 0, value: ''}],
+    textboxes: this.props.children || [{id: 0, value: ''}],
```

Comment:
This seems weird, why is this necessary?

Reply:
The children of a React component should be React nodes — things that will be rendered in the DOM. This array of data is essentially the default value of the form control (`TextboxRepeater`), so it should be passed in as an explicit prop like `defaultValues`.